packages/toolkit/src/view/knowledge/components/CreateKnowledgeDialog.tsx

```diff
@@ -0,0 +1,243 @@
+                    <Form.Label className="text-semantic-fg-primary product-button-button-2">
+                      Knowledge base name
+                    </Form.Label>
+                    <Form.Control>
+                      <Input.Root>
+                        <Input.Core
+                          {...field}
+                          id={field.name}
+                          placeholder="Knowledge base name"
+                        />
+                      </Input.Root>
+                    </Form.Control>
+                    <div className="h-6">
+                      {nameValue && !isNameValid && (
+                        <p className="!mt-0.5 text-semantic-fg-secondary product-body-text-4-regular">
+                          Name will be transformed to: {formattedName}
+                        </p>
+                      )}
+                      <Form.Message className="!mt-0.5 product-body-text-4-regular" />
+                    </div>
+                  </Form.Item>
+                )}
+              />
+            </div>
+
+            <Form.Field
+              control={form.control}
+              name="description"
+              render={({ field }) => (
+                <Form.Item className="!space-y-1">
+                  <Form.Label className="mb-1 text-semantic-fg-primary product-button-button-2">
+                    Description
+                  </Form.Label>
+                  <Form.Control>
+                    <Textarea
+                      {...field}
+                      id={field.name}
+                      placeholder="Content"
+                    />
+                  </Form.Control>
+                  <p className="text-semantic-fg-secondary product-body-text-4-regular">
+                    Fill with a short description
+                  </p>
+                  <Form.Message />
+                </Form.Item>
+              )}
+            />
+            {/* <Form.Field
+              control={form.control}
+              name="tags"
+              render={({ field }) => (
+                <Form.Item>
+                  <div className="flex items-center justify-between">
+                    <Form.Label className="text-semantic-fg-primary product-button-button-2">
+                      Tags
+                    </Form.Label>
+                    <p className="my-auto text-semantic-fg-secondary product-body-text-4-regular">
+                      Optional
+                    </p>
+                  </div>
+                  <Form.Control>
+                    <Input.Root>
+                      <Input.Core
+                        {...field}
+                        id={field.name}
+                        placeholder="Add tag"
+                      />
+                    </Input.Root>
+                  </Form.Control>
+                  <Form.Message />
+                </Form.Item>
+              )}
+            /> */}
+            <div className="mt-8 flex justify-end gap-x-3">
```

Comment:
@thewbuk I will encourage you to use one direction margin only, and in most case in this repo, use margin-bottom